File: com_tower/pkg/tower/tower.go
package tower

import (
	"github.com/ViniiSouza/maritime_flow/com_tower/pkg/utils"
)

type Tower struct {
	UUID      utils.UUID `json:"tower_uuid" db:"id"`
	Latitude  float64    `json:"latitude" db:"latitude"`
	Longitude float64    `json:"longitude" db:"longitude"`
}

type TowerHealthRequest struct {
	Id utils.UUID `json:"tower_id"`
}

type TowersPayload struct {
	Towers []Tower `json:"towers"`
}

File: mobility_core/MobilityCore/MobilityCore.Shared/Models/Tower.cs
using System.Text.Json.Serialization;

namespace MobilityCore.Shared.Models;

public class Tower
{
    [JsonPropertyName("tower_uuid")]
    public string TowerUuid { get; set; } = string.Empty;
    
    [JsonPropertyName("latitude")]
    public double Latitude { get; set; }
    
    [JsonPropertyName("longitude")]
    public double Longitude { get; set; }
}

